deploy/eks/action.yml

```diff
@@ -34,4 +37,12 @@ runs:
     - name: Deploying to EKS via Helm release
       shell: bash
       run: |
-        chmod +x ./deploy/build.sh && ./deploy/build.sh service deploy_eks --env=${{ inputs.ENV }}
+        cmd="chmod +x ./deploy/build.sh && ./deploy/build.sh service deploy_eks --env=${{ inputs.ENV }}"
+        
+        # Check if timeout is set and provide it as a flag
+        if [ -n "${{ inputs.TIMEOUT }}" ]; then
+          cmd="${cmd} --timeout=${{ inputs.TIMEOUT }}"
+        fi
```

Comment:
We could replace this if we instead use a default value (same as we do with inputs.ENV above), but I would rather not do that. Currently, the default timeout value on engCaps side is set to `5m` but if we define our own here, it may become disjointed